src/app/app-routing.module.ts

```diff
@@ -8,6 +8,7 @@ import { SearchPlaygroundComponent } from './search-playground/search-playground
 import { SearchResultsComponent } from './search-playground/search-results/search-results.component';
 import { StillImagePlaygroundComponent } from './still-image/still-image-playground.component';
 import { ViewerPlaygroundComponent } from './viewer-playground/viewer-playground.component';
+import { CkeditorPlaygroundComponent } from './ckeditor/ckeditor-playground/ckeditor-playground.component';
```

Comment:
organize imports

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808